cmake/Modules/CocosConfigDefine.cmake

```diff
@@ -166,19 +166,6 @@ endfunction()
     if(MSVC)
         target_compile_options(${target}
             PUBLIC /MP
-            PUBLIC /Z7
-            PUBLIC /MD$<$<CONFIG:Debug>:d>
         )
-    else()
-        target_compile_options(${target} 
-            PUBLIC -Wall
```

Comment:
`-Wall` is one of the the defaults values, too?

Reply:
In modern cmake flags should not be set inside library. Please see for example:
https://gist.github.com/mbinna/c61dbb39bca0e4fb7d1f73b0d66a4fd1#dont-abuse-usage-requirements

We may add this to root folder but it should not be set here (and definitely not as PUBLIC).